src/modules/globals.ts

```diff
@@ -28,8 +30,8 @@ type TmpGameType = {
     wallet: Wallet,
     keyItems: any,
     badgeCase: BadgeCase,
-    oakItems: any,
-    oakItemLoadouts: any,
```

Comment:
I knew this file existed somewhere... was trying to find it for that react pr instead of the GameShim I made...